lib/utils/downloadTemplateFromRepo.js

```diff
@@ -46,6 +47,14 @@ function validateUrl({ url, hostname, service, owner, repo }) {
   }
 }
 
+/**
+ * Check if the URL is pointing to a Git repository
+ * @param {String} url
+ */
+function isPlainGitURL(url) {
+  return url.endsWith('.git');
```

Comment:
Nothing significant but we may double ensure it's `https` (afaik git repo urls can only work via https)